CenterForOpenScience/osf.io
# Feature/APIv2 Restore Meeting Download Counts Sorting [ENG-667]

❗️ Make sure page counter tables are populated on every env

## Purpose
Performance of sorting APIv2 MeetingSubmission download counts was poor with old PageCounter table. https://github.com/CenterForOpenScience/osf.io/pull/8800 updated the PageCounter model to more properly it to the resource and the file.  Now we can query on these new columns to make the download count queries more efficient.  


## Changes
Restores sorting on meeting submission download counts by querying on new PageCounter columns.

## QA Notes

- Make meeting submissions, add files, with different numbers of download counts. Verify sorting works in API. (Ember-side change will have to be made to restore UI sort buttons, but you can verify independently in the API).

## Documentation



## Side Effects



## Ticket

https://openscience.atlassian.net/browse/ENG-667
